_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -215,7 +216,7 @@ These two SVG `a` and HTML `a` elements have the same [accessible name][], same
 
 #### Failed Example 1
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context] but go to different resources.
+These two HTML `a` elements have the same [accessible name][], [accessible description][] and [context][programmatically determined link context] but go to different resources.
```

Comment:
I think we still have the problem where this example is falling into the "ambiguous to users in general" exception of 2.4.4 and therefore shouldn't fail, but this is an unrelated problem.